upcloud/storage.go

```diff
@@ -55,8 +55,8 @@ const (
 	StorageImportStateCancelled  = "cancelled"
 	StorageImportStateCompleted  = "completed"
 
-	StorageEncryptionDataAtReset StorageEncryption = "data-at-rest"
-	StorageEncryptionNone        StorageEncryption = "none"
+	StorageEncryptionDataAtRest StorageEncryption = "data-at-rest"
```

Comment:
Another option would be to mark the one with typo as deprecated and just add the fixed one in addition to that 🤔

Reply:
I would go with the deprecation process, even though it's only a small change.